src/client/client.js

```diff
@@ -49,6 +45,9 @@ const createCashay = (store, cashaySchema) => {
     // ENABLE THIS FOR EXPLORING FRONT END PERFORMANCE
     // const {whyDidYouUpdate} = require('why-did-you-update');
     // whyDidYouUpdate(React);
+    // ENABLE THIS AS ANOTHER MEANS OF TESTING FRONT END PERFORMANCE
+    // const Perf = require('react-addons-perf');
+    // window.Perf = Perf;
```

Comment:
I am seeing some performance degradation on occasion. I still don't know what's causing it. I appears unrelated to this change.